quandl/operations/data_list.py

```diff
@@ -17,5 +17,18 @@ def create_list_from_response(cls, data):
         return DataList(cls, values, metadata)
 
     @classmethod
+    def create_datatable_list_from_response(cls, data):
+        if len(data['datatable']['data']) > 0 \
+                and len(data['datatable']['columns']) != len(
+                    data['datatable']['data'][0]):
+            raise InvalidDataError(
+                'number of column names does not match number of data points in a row!',
```

Comment:
When could this type of error occur? And if it does we should tell @lynnyen88 we need a translation.